lib/pengine/group.c

```diff
@@ -23,6 +23,56 @@
 #define VARIANT_GROUP 1
 #include "./variant.h"
 
+/*!
+ * \internal
+ * \brief Check whether a group flag is set
+ *
+ * \param[in] group  Group resource to check
+ * \param[in] flags  Flag or flags to check
```

Comment:
might specify the flag type here and below, but the use case is so limited (and deprecated) it doesn't really matter